dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
I may be misunderstanding how the pieces fit together, the easiest thing is probably to run a small file through the scigraph OWL Loader and evaluate the output.